src/components/Icon/Icon.mdx

```diff
@@ -27,15 +29,39 @@ All icons have their own component. Each component name is a pascal cased versio
   <X />
 </Playground>
 
-## Changing icon size
+### Styling using CSS in JS
+
+There are a few ways this can be done. The default styles can be changed using a theme.
+
+To apply custom styles using CSS in JS, you can use the styles property.
+
+<Playground>
+  <GitHub />
+  <GitHub style={{width: '24px', height: '24px', stroke: '#116ab8'}}/>
```

Comment:
I prefer having a size prop. It's a grey zone. I say revert

Reply:
I tried. Our CSS seems to be overwriting this property value.